mainpage.css

```diff
@@ -0,0 +1,192 @@
+/** 
+  * mainpage.css
```

Comment:
Do we need the file name up here?

Reply:
@jantnovi I don't think it is actually necessary or mandatory but from the CSS best practices research I've done, it is encouraged to do so. I'm not too sure actually.

Reply:
eh seems redundant to me

Reply:
Removed

Reply:
am I crazy or did you guys not remove these redundant names